### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by sequencemedialimited**
- Added `.npmrc` to `.gitignore`
- Added `.nvmrc` to `.gitignore`
- Removed Wallaby config
- Amended "description"
- Amended "author"
- Updated `devDependencies`

### CODE REVIEW

**Comment by sequencemedialimited** [comment]
I will lint in another PR (there are _lots_ of errors with the upgrade to `@ministryofjustice/eslint-config-fb` version `1.1.2`)

I'll update `dependencies` and increment the package version afterward

File: package.json [hunk 1]
```diff
@@ -4,27 +4,25 @@
...
-  "author": "github@alex.fu2k.org",
+  "author": "Form Builder Team <form-builder-team@digital.justice.gov.uk>",
   "license": "MIT",
   "devDependencies": {
-    "@ministryofjustice/eslint-config-fb": "^1.0.7",
-    "tape": "^4.11.0"
+    "@ministryofjustice/eslint-config-fb": "^1.1.2"
   },
   "dependencies": {
+    "@ministryofjustice/fb-components-core": "^3.1.0-7",
```
Comment (asmega) [inline package.json]: this is outdated now i believe. runner now loads the library so this this line is not needed. can you rebase with master please to make sure

Comment (sequencemedialimited) [reply package.json]: Good spot

Comment (sequencemedialimited) [reply package.json]: Line gone

**Pull request merged**